// ngx-file-drop/esm2020/ngx-file-drop/ngx-file-drop.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { NgxFileDropComponent } from './ngx-file-drop.component';
import { NgxFileDropContentTemplateDirective } from './ngx-templates.directive';
import * as i0 from "@angular/core";
export class NgxFileDropModule {
}
NgxFileDropModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.0.0", ngImport: i0, type: NgxFileDropModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
NgxFileDropModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.0.0", ngImport: i0, type: NgxFileDropModule, bootstrap: [NgxFileDropComponent], declarations: [NgxFileDropComponent,
        NgxFileDropContentTemplateDirective], imports: [CommonModule], exports: [NgxFileDropComponent,
        NgxFileDropContentTemplateDirective] });
NgxFileDropModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.0.0", ngImport: i0, type: NgxFileDropModule, providers: [], imports: [CommonModule] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.0.0", ngImport: i0, type: NgxFileDropModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        NgxFileDropComponent,
                        NgxFileDropContentTemplateDirective,
                    ],
                    imports: [
                        CommonModule
                    ],
                    exports: [
                        NgxFileDropComponent,
                        NgxFileDropContentTemplateDirective,
                    ],
                    providers: [],
                    bootstrap: [
                        NgxFileDropComponent
                    ],
                }]
        }] });
